components/layouts/blog/blog-view.hbs

```diff
@@ -91,7 +91,7 @@
     </div> <!-- .two-col-content -->
 
 
-    <div class="grid-col-content__three section__inner">
+    <div class="grid-col-content__three-blog section__inner">
```

Comment:
Just a note for later cleanup, we have a better block we could do a modifier on in the `columns` block, so we could change this to be `columns--three-blog` to be consistent with all the others. No need to do it now since this is fully baked, but let's catch it up down the road (or maybe in a swarm).